create_builder.go

```diff
@@ -103,6 +103,14 @@ func (c *Client) CreateBuilder(ctx context.Context, opts CreateBuilderOptions) e
 }
 
 func validateBuildpack(bp dist.Buildpack, source, expectedID, expectedBPVersion string) error {
+	if bp == nil {
+		return fmt.Errorf(
+			"buildpack not set from %s. should have ID %s",
+			style.Symbol(source),
+			style.Symbol(expectedID),
+		)
+	}
+
```

Comment:
Validate that bp is not unset, which would cause panics when calling interface methods.